source/connect.txt

```diff
@@ -13,13 +13,75 @@ Connect to a Deployment
    :class: singlecol
 
 This page shows how to use the |mdb-shell| to connect to a MongoDB
-deployment.
+deployment. You can connect to a 
+`MongoDB Atlas cloud-hosted deployment <https://www.mongodb.com/docs/atlas>`__,
+connect to a local deployment, or connect to another remote host with
+|mdb-shell|.
 
 Prerequisites
 -------------
 
 .. include:: /includes/fact-connect-prereqs.rst
-   
+
+Install ``mongosh``
+~~~~~~~~~~~~~~~~~~~
+
+These procedures assume you have already installed ``mongosh``. For more 
+information about installing ``mongosh``, refer to 
+:ref:`mdb-shell-install`.
+
+Connect to a MongoDB Atlas Deployment
+-------------------------------------
+
+You can connect to your MongoDB Atlas deployment directly from your shell.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Get your Atlas Connection String
+
+      You need an Atlas connection string to connect from |mdb-shell|.
+      You can get the Atlas connection string in the Atlas UI.
+
+      Refer to the
+      `Find Your MongoDB Atlas Connection String 
+      <https://www.mongodb.com/docs/manual/reference/connection-string/#find-your-mongodb-atlas-connection-string>`__ 
+      guide for details.
+
+   .. step:: Set Your Database Credentials
+
+      If you haven't already 
+      `established a database user 
+      <https://www.mongodb.com/docs/atlas/tutorial/create-mongodb-user-for-cluster/>`__, 
+      you must set a username and password. You pass the username with 
```

Comment:
[nit/suggestion] for simplicity/brevity
```suggestion
      you must set a username and password. Pass the username with 
```

Reply:
[suggestion] Also based on our [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#:~:text=Sentences%20that%20describe%20a%20condition%20should%20always%20begin%20with%20the%20conditional%20clause%2C%20so%20that%20the%20user%20can%20skip%20the%20section%20if%20the%20condition%20does%20not%20apply%20to%20them.), we begin conditional statements with the conditional clause.
```suggestion
      you must set a username and password. To connect to Atlas, pass your username with the connection string.
```